### PR TITLE
Create yaml for US Sen Hawley

### PR DESCRIPTION
**Pull request by OdeclasV**

### CODE REVIEW

File: members/FMOS63398500.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          Mr. and Mrs.: Mr. and Mrs.
+          Dr.: Dr.
+          Dr. and Mrs.: Dr. and Mrs.
+          Professor: Professor
+      - name: Topic
+        selector: "#edit-topic"
+        value: $TOPIC
+        required: true
+        options:
+          Civil Rights: Civil Rights
```
Comment (tomdionesotes) [inline members/FMOS63398500.yaml]: can you flesh this out a bit more?

File: members/FMOS63398500.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        value: $NAME_LAST
+        required: true
+      - name: Street
+        selector: "#edit-address-1"
+        value: $ADDRESS_STREET
+        required: true
+      - name: Street2
+        selector: "#edit-address-2"
+        value: $ADDRESS_STREET_2
+        required: true
```
Comment (tomdionesotes) [inline members/FMOS63398500.yaml]: street2 is `required`?

Comment (OdeclasV) [reply members/FMOS63398500.yaml]: good catch, gracias

File: members/FMOS63398500.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/FMOS63398500.yaml]: added rest of topic options

**Pull request merged**